HeartbeatLogging/Sources/HeartbeatController.swift

```diff
@@ -41,17 +52,48 @@ public final class HeartbeatController {
   ///
   /// - Parameter info: A `String` identifier to associate a new heartbeat with.
   public func log(_ info: String) {
-    let newHeartbeat = Heartbeat(info: info)
-    storage.offer(newHeartbeat)
+    let (agent, date, capacity) = (info, dateProvider(), limit)
```

Comment:
Optional. Is it same as just assigning values of the variables on separate lines like `let agent = info`? If so, using 3 separate lines for it looks more readable to me.

Reply:
Reworked in e9d0361